components/FAQ/faq.tsx

```diff
@@ -0,0 +1,23 @@
+import Link from 'next/link';
+import { FC, PropsWithChildren } from 'react';
+import styles from './styles.module.scss';
+
+type Props = {
+  link: string;
+};
+
+const FAQ: FC<PropsWithChildren<Props>> = ({ children, link }) => (
```

Comment:
FC<Props>, because 
`type FC<P = {}> = FunctionComponent<P>;

    interface FunctionComponent<P = {}> {
        (props: PropsWithChildren<P>, context?: any): ReactElement<any, any> | null;
        propTypes?: WeakValidationMap<P> | undefined;
        contextTypes?: ValidationMap<any> | undefined;
        defaultProps?: Partial<P> | undefined;
        displayName?: string | undefined;
    }`

Reply:
fixed